static/src/javascripts/projects/common/modules/ui/last-modified.js

```diff
@@ -4,14 +4,19 @@ import fastdom from 'lib/fastdom-promise';
 
 const lastModified = (): void => {
     fastdom
-        .read(() => document.getElementsByClassName('js-lm')[0])
-        .then(jsLm => {
+        .read(() => ({
+            jsLm: document.getElementsByClassName('js-lm')[0],
+            jsWpd: document.getElementsByClassName('js-wpd')[0],
```

Comment:
If the element doesn't exist, this will throw an exception. It's safer to pass the whole array and then test the `length` attribute in both cases I reckon.